quick_search-archives_space_searcher.gemspec

```diff
@@ -0,0 +1,22 @@
+$LOAD_PATH.push File.expand_path('lib', __dir__)
+
+# Maintain your gem's version:
+require 'quick_search/archives_space_searcher/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = 'quick_search-archives_space_searcher'
+  s.version     = ArchivesSpaceSearcher::VERSION
+  s.authors     = ['Chris Fitzpatrick']
```

Comment:
Please update to reflect "UMD Libraries" authorship, email address and home page (see https://github.com/umd-lib/quick_search-swiftype_searcher/blob/develop/quick_search-swiftype_searcher.gemspec)